### PR TITLE
Support MosOS Linux distribution

### LINKED ISSUE #155: Support MosOS Linux distribution

### PR DESCRIPTION
**Pull request by elhimov**
Closes #155

### CODE REVIEW

File: pack/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ else ifneq (,$(wildcard /etc/debian_version))
 include $(PACKDIR)/deb.mk
 else ifneq (,$(wildcard /etc/alpine-release))
 include $(PACKDIR)/apk.mk
-else ifneq (,$(shell grep "^ID=\"opensuse-leap\"" /etc/os-release))
+else ifneq (,$(shell grep "opensuse" /etc/os-release))
```
Comment (Totktonada) [inline pack/Makefile:17]: It would be nice if the commit (and the pull request description) reveals details about the target OS and the needed changes. At least I would like to know a content of the `/etc/os-release` file on this OS.

File: pack/rpm.mk [hunk 2]
```diff
@@ -20,7 +20,7 @@ endif
 
 # Usual 'Release' RPM spec directive value is 1%{?dist}, where
 # where 1 is $(RELEASE) value and %{dist} is like .el8 or .fc31.
-RPMRELEASE := $(RELEASE)%{dist}
+RPMRELEASE := $(RELEASE)%{?dist}
```
Comment (Totktonada) [inline pack/rpm.mk:23]: Same here. It is not clear why the change is needed. I would appreciate some explanation in the commit message/pull request description.

At least, I understand it as follows.

The given OS doesn't define the `dist` macro. It is unclear, whether a suffix like `.el7` or `.fc35` exists for packages provided by the OS repositories. If so, we should add it.

File: pack/Makefile (hunk 1 above)
Comment (elhimov) [reply pack/Makefile]: Fixed (added separate condition to check against MosOS-Arbat distribution).

File: pack/rpm.mk (hunk 2 above)
Comment (elhimov) [reply pack/rpm.mk]: Fixed (added separate section that follows the approach used in openSUSE, but using prefix 'at' instead of 'lp')

Comment (elhimov) [reply pack/rpm.mk]: Regarding 'at' prefix: it's a kind of heuristic. I just checked several packages on MosOS Arbat 15.4 and found out that many of them have 'at154' in their names in the same way as 'lp150/lp151/lp152' in openSUSE of versions 15.0, 15.1, 15.2.

File: pack/Makefile (hunk 1 above)
Comment (Totktonada) [reply pack/Makefile]: Now it looks quite straightforward, I'm OK. Thanks!

File: pack/rpm.mk (hunk 2 above)
Comment (Totktonada) [reply pack/rpm.mk]: Thanks!

**Review by Totktonada — APPROVED** [review]
